website/docs/r/elasticache_replication_group.html.markdown

```diff
@@ -88,6 +88,9 @@ resource "aws_elasticache_replication_group" "baz" {
 }
 ```
 
+~> **Note:** While parameter_group_name is optional not setting it will result in creation of a cluster with ClusterMode off.
+This then results in forced recreation of the resource as AWS will return ClusterEnabled false while Terraform has it set to enabled in the terraform state.
```

Comment:
Maybe instead/in addition you could add the following note below near the `Cluster Mode`:

> Note, if enabling cluster mode via this block, be sure to use a parameter group that includes `cluster-enabled=yes`, for example `default.redis5.0.cluster.on`.  Otherwise, the replication group resource will be recreated each time you apply.